devtools/gradle/src/main/java/io/quarkus/gradle/GradleDependencyArtifact.java

```diff
@@ -0,0 +1,117 @@
+package io.quarkus.gradle;
+
+import org.gradle.api.artifacts.DependencyArtifact;
+
+public class GradleDependencyArtifact implements DependencyArtifact {
```

Comment:
why the need for a different one ?

Reply:
What's the alternative?

Reply:
ah, i read this as extending Dependency not implementing.